README.md

```diff
@@ -304,7 +304,31 @@ try (DatabaseReader reader = new DatabaseReader.Builder(database).build()) {
     System.out.println(response.isResidentialProxy()); // false
     System.out.println(response.isTorExitNode()); //true
 }
+```
+
+### Anonymous Plus ###
+
+```java
+// A File object pointing to your GeoIP2 Anonymous IP database
```

Comment:
Some of the comments reference `GeoIP2 Anonymous IP`, and I wonder if some of the (such as this one) should instead reference `GeoIP2 Anonymous Plus`